src/NzbDrone.Common/Extensions/UrlExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace NzbDrone.Common.Extensions
+{
+    public static class UrlExtensions
+    {
+        public static bool IsValidUrl(this string path)
+        {
+            if (string.IsNullOrWhiteSpace(path))
```

Comment:
yeah, mainly, it's more explicit about where the if boundaries start and end, leaves less room for mistakes.